lib/contents.php

```diff
@@ -97,7 +97,18 @@ function getContents($url, $header = array(), $opts = array()){
 	// Enables logging for the outgoing header
 	curl_setopt($ch, CURLINFO_HEADER_OUT, true);
 
-	$data = curl_exec($ch);
+	// Use file_get_contents if in CLI mode with no root certificates defined
+	if(php_sapi_name() == 'cli' && ini_get('curl.cainfo') == '') {
+		$data = file_get_contents($url);
+		if($data)
+			$errorCode = 200;
+		else
+			$errorCode = 500;
```

Comment:
Please use curly braces `{}` for the if-statement above or make it a single line if-statement:
 `$errorCode = ($data === false) ? 500 : 200;`

_Note_: $data may return a value that can be interpreted as `false`, so the check should be explicit.